backend/server.js

```diff
@@ -127,6 +127,106 @@ async function requestHandler(request, response) {
+            case 1:
+                await remindStoreOwner(package);
+                break;
+            default:
+                break;
+        }
+    }
+}
+
+/* Sends a reminder to the customer associated with the package if it more than 3 days old and isn't booked for pickup yet */
+async function sendReminder(package) {
+    const msPerDay = 86400000;
+    const days = 3;
+    let now = new Date();
+    let creationDelta = now-package.creationDate;
+
+    if(creationDelta >= msPerDay*days) {
+        console.log('Sending reminder to: ' + package.customerEmail + ' (3 days has passed)');
+        sendEmail(package.customerEmail, package.customerName, "Reminder: no time slot booked", `Link: http://127.0.0.1:8000/package/${package.guid}/select_time`, await reminderHTML(package));
+        /* Increment package.remindersSent in database */
+        db.run("UPDATE package SET remindersSent=1 WHERE id=?", [package.id]);
+    } else {
+        return;
+    }
+}
+
+/* Sends a reminder to the store owner associated with the package if it more than 14 days old and isn't booked for pickup yet */
+async function remindStoreOwner(package) {
+    const msPerDay = 86400000;
+    const days = 14;
+    let now = new Date();
+    let creationDelta = now-package.creationDate;
+    let store = await storeIdToStore(package.storeId);
+
+    if(creationDelta >= msPerDay*days) {
+        console.log('Sending reminder to store owner: ' + store.storeEmail + ' (14 days has passed - order: ' + package.externalOrderId + ')');
+        sendEmail(store.storeEmail, store.name, "Reminder: no time slot booked", `Order: ${package.externalOrderId}`, await reminderStoreHTML(package));
+        /* Increment package.remindersSent in database */
+        db.run("UPDATE package SET remindersSent=2 WHERE id=?", [package.id]);
+    } else {
+        return;
+    }
+}
+
+async function reminderStoreHTML(package) {
+    return `
+        <html>
+            <head>
+                <meta charset="UTF-8">
+            </head>
+            <body>
+                <h1>Unbooked time slot</h1>
+                <p>Order: ${package.externalOrderId}</p>
+                <p>Customer info:<br>${package.customerName} (${package.customerEmail})</p>
+                <p>Created: ${new Date(package.creationDate)}</p>
+            </body>
+        </html>
+    `
+}
+
+async function reminderHTML(package) {
+    let store = await storeIdToStore(package.storeId);
+
+    return `
+        <html>
+            <head>
+                <meta charset="UTF-8">
+            </head>
+            <body>
+                <h1>Unbooked time slot</h1>
+                <p>Hello ${package.customerName}, you still have not picked a time slot for picking up your order from ${store.name}</p>
+                <p>Follow this link to book a time slot:</p>
+                <a target="_blank" href="http://127.0.0.1:8000/package/${package.guid}/select_time">http://127.0.0.1:8000/package/${package.guid}/select_time</a>
```

Comment:
og igen er det rigtige link `http.//127.0.0.1:8000/package?guid={guid}` selve addressen `http://127.0.0.1:8000` kunne måske gemmes i en konstant istedet for